### DISCUSSION TITLE
tsconfig.json anpassen

### DISCUSSION THREAD

**Discussion by Philipp1L2P** [VDI-CodING/codeING-main#27] [Q&A] [answered]
Der Ausschnitt muss ab **Zeile 70** bei mir eingefügt werden:

```
 },
   "exclude": [
      "node_modules",
      "./build"
   ]
}
```

**Comment by stan1025** [comment, marked as answer]
Wie sich im Kickoff gezeigt hat, variiert die Zeilennummer bei einigen, dass die im Tutorial stehenden Zeilennummern nicht stimmen.
Faktisch sind einfach die letzten beiden schließenden Klammern '}' mit dem Ausschnitt zu ersetzen.
